enterprise/web.jsf/src/org/netbeans/modules/web/jsf/palette/JSFPaletteUtilities.java

```diff
@@ -242,7 +240,7 @@ protected static ScriptEngine getScriptEngine(FileObject fo) {
                     } catch (ClassNotFoundException ex) {
                         Exceptions.printStackTrace(ex);
                     }
-                    manager = new ScriptEngineManager(loader != null ? loader : Thread.currentThread().getContextClassLoader());
```

Comment:
Why is loader ignored here?

Reply:
The classloader logic is handled centrally in `Scripting.createManager()`.